doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -555,6 +555,14 @@ Libraries for NFC
 
 |no_changes_yet_note|
 
+  * :file:`ncs/nrf/subsys/nfc/lib/platform.c`:
+
+    * Updated:
+
+      * Add the possibility of deffering nfc callback to a thread context. (Experimental)
+      * Add support for Zero Latency Interrupts for NFC. (Experimental)
+      * Align file with new library implementation.
```

Comment:
```suggestion
      * Added the possibility of deferring a NFC callback to a thread context. (Experimental)
      * Added support for Zero Latency Interrupts for NFC. (Experimental)
      * Aligned the :file:`ncs/nrf/subsys/nfc/lib/platform.c` file with new library implementation.
```
Maybe "move" instead of "defer"?
Do we need to capitalize zero latency interrupt?

Reply:
I checked. This phrase does not need capitalization